packages/terra-arrange/package.json

```diff
@@ -45,6 +45,7 @@
     "terra-mixins": "^1.0.0"
   },
   "devDependencies": {
-    "terra-toolkit": "^0.x"
+    "terra-toolkit": "^0.x",
+    "terra-icon": "^0.x"
   }
 }
```

Comment:
The nightwatch test scripts need to be added in here.
Example: https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L36